Exercise-HTML-CSS-1/css/styles.css

```diff
@@ -0,0 +1,67 @@
+* {
```

Comment:
If you use Bootstrap and you don't have any other custom font you want to use, you should not overwrite font-family. Specially in such a general rule